packages/dashboard/src/components/playground/drawerHeader/drawer-header.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from '@cloudscape-design/components';
+import React from 'react';
+import {
+  spaceScaledXs,
+  spaceScaledXxxs,
+} from '@cloudscape-design/design-tokens';
+import './drawer.css';
+export const DrawerHeader = ({
```

Comment:
There are a few formatting inconsistencies in this change, including this (lack of newline) and double newlines in other places.

Reply:
i ran npm run lint, guess something in eslint is broken ?

Reply:
Might not test newline behavior.